.ci/build-mac.sh

```diff
@@ -40,7 +42,15 @@ export CPPFLAGS="-I$BREW_X64_PATH/include -msse -msse2 -mcx16 -no-pie"
 export LIBRARY_PATH="$BREW_X64_PATH/lib"
 export LD_LIBRARY_PATH="$BREW_X64_PATH/lib"
 
-git submodule update --init --recursive --depth 1
+export VULKAN_SDK
+VULKAN_SDK="$BREW_X64_PATH/opt/molten-vk"
+ln -s "$VULKAN_SDK/lib/libMoltenVK.dylib" "$VULKAN_SDK/lib/libvulkan.dylib"
+export VK_ICD_FILENAMES="$VULKAN_SDK/share/vulkan/icd.d/MoltenVK_icd.json"
+
+export LLVM_DIR
+LLVM_DIR="BREW_X64_PATH/opt/llvm@16"
+# exclude FAudio, SPIRV, moltenvk and LLVM from submodule update
+git submodule -q update --init --depth=1 --jobs=8 $(awk '/path/ && !/FAudio/ && !/MoltenVK/ && !/llvm/ && !/SPIRV/ { print $3 }' .gitmodules)
```

Comment:
I don't really understand the purpose of ignoring MoltenVK here.
We don't even have it in our gitmodules file

Reply:
to prevent downloading and trying to compile it , to use the prebuilt moltenvk